firmware.ino

```diff
@@ -65,7 +77,13 @@ void handleCommand(Command c) {
   log("Cmd type: ");
   logint((int)c.commandType);
   logln("");
+  Command pong = Command(c.commandData, PONG, thisCell, HARDWARE); // putting the line here works
   switch (c.commandType) {
+    case PING:
+      logln("PINGU");
+      //Command pong = Command(c.commandData, PONG, thisCell, HARDWARE); // putting it here doesn't
```

Comment:
Here's the "delightfully fun" bit. The top line works, if I replace it with the commented-out one, it doesn't go figure.
If anyone knows why, feel free to fix.